src/jit/SimdArm32Inl.h

```diff
@@ -560,7 +565,7 @@ static void emitBinarySIMD(sljit_compiler* compiler, Instruction* instr)
     simdOperandToArg(compiler, operands + 1, args[1], type, SLJIT_FR2);
 
     args[2].set(operands + 2);
-    sljit_s32 dst = GET_TARGET_REG(args[2].arg, SLJIT_FR0);
+    sljit_s32 dst = GET_TARGET_REG(args[2].arg, isSwizzle ? SLJIT_FR4 : SLJIT_FR0);
```

Comment:
FR4? What are the lower indicies used?

Reply:
The 1. operand -> FR0, FR1
The 2. operand -> FR2, FR3